contents/DashboardBenefitCardConstants.js

```diff
@@ -79,6 +82,9 @@ const ACTIVE_OAS = {
       description: 'July 1, 2021',
     },
   ],
+  editAddressLink: t.oasBenefitEditAddressLink,
```

Comment:
I don't think these will be passed in as part of the APIs in order for them to be a part of the contents

Reply:
they won't be but if edit address link is different for each benefit then we need to pass it as prop